internal/driver/driver_focus.go

```diff
@@ -98,19 +123,33 @@ func compileTagFilter(name, value string, ui plugin.UI, err error) (func(*profil
 		}
 		rfx = append(rfx, fx)
 	}
+	if wantKey == "" {
+		return func(s *profile.Sample) bool {
+		matchedrx:
+			for _, rx := range rfx {
+				for key, vals := range s.Label {
+					for _, val := range vals {
+						if rx.MatchString(key + ":" + val) {
```

Comment:
Add a TODO that adding "key:" into the matched string will be going away some day?